packages/valory/skills/abstract_round_abci/skill.yaml

```diff
@@ -34,6 +34,7 @@ connections:
 contracts:
 - valory/service_registry:0.1.0:bafybeidtwbj7mwiskbbmsw43zzooivuq6nripdzn7yuoou66rq7spom54m
 protocols:
+- open_aea/signing:1.0.0:bafybeid6f5ceool4evgaxdegs5pkjoivqjk3mo3ehctalswngb5m5c32wm
```

Comment:
There are further protocol inconsistencies. None of them crucial as the higher or lower level dependencies reference the relevant packages. But an issue has been opened to resolve this properly: https://github.com/valory-xyz/open-aea/issues/247